src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int WORD_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
[Split program logic into at least 3 different methods and call them line by line in `getStatistic()`:](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic)
1. readFromFile - method should read all data from the file and return String([use StringBuilder to concat data from file](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#use-stringbuilder-instead-of-string-to-concat-data-in-loops))
2. createReport - method should count buy, supply, and result and return the report as String 
3. writeToFile - method should write your report(String) to the file